.gitignore

```diff
@@ -32,3 +32,5 @@ snprc_ehr/.editorconfig
 snprc_ehr/resources/referenceStudy/LoadTaqmanData
 snprc_ehr/resources/referenceStudy/datasetImport/LoadTaqmanData
 snprc_scheduler/.gradle
+snprc_ehr/resources/credits
```

Comment:
What file is this? Generated locally?

Reply:
This is the directory that has the dependencies.txt file generated in it - I think it's getting generated when the apache-commons-text lib gets built.